src/Utilities/brand.js

```diff
@@ -0,0 +1,62 @@
+export const getBrand = (data, roadster, nautica,puma,adidas,hAndm,colorbar,klotthe,talesAndStories,dove) => {
+
+  let filteredBrand = [];
+  if (roadster === false && nautica === false) {
+    return data;
+  }
+  if (roadster) {
+    let newList = data.filter(
+      (product) => product.brand.toLowerCase() === "roadster"
+    );
+    filteredBrand.push(...newList);
+  }
+  if (nautica) {
```

Comment:
again ! just use a `loop` to check in the array and if the `array` is populated with that particular brand it will render